Missing $ in metavariable SOURCE.

infractal opened the pull request.

infractal committed:
Missing $ in metavariable SOURCE.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/returntocorp/semgrep-rules?pullRequest=2892) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/returntocorp/semgrep-rules?pullRequest=2892) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/returntocorp/semgrep-rules?pullRequest=2892) it.</sub>

infractal committed:
Added addBatch / nativeSQL / create(.*) / prepare as potential sink methods for sqli
  (execute(), executeQuery(), executeUpdate() already covered by regex
Statement sink methods:
  execute, executeQuery, executeUpdate, ...
EntityManager sink methods:
  createNativeQuery, createQuery, createStoredProcedureQuery
Connection sink methods:
  nativeSQL, prepareCall

infractal committed:
Add prepared statements (can be used incorrectly) as sinks.

infractal committed:
Handle string concats + mutable StringBuffer/StringBuilder as propagators.
(does not handle sbf.append().append() ... )
